package/gluon-core/i18n/de.po

```diff
@@ -0,0 +1,47 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Project-Id-Version: PACKAGE VERSION\n"
+"PO-Revision-Date: 2025-05-04 23:56+0200\n"
+"Last-Translator: maurer <fmaurer@disroot.org>\n"
+"Language-Team: German <translation-team-de@lists.sourceforge.net>\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid "Autoupdater branch"
+msgstr "Autoupdater-Kanal"
```

Comment:
In gluon-web-autoupdater übersetzen wir "branch" mit "Branch" - zur Konsistenz würde ich hier also auch "Autoupdater-Branch" vorziehen.